misc/packaging/gen_acrn_deb.py

```diff
@@ -226,6 +226,23 @@ def create_configurator_deb(version, build_dir):
             dest.write(src.read())
     return
 
+def clean_configurator_deb(version, build_dir):
+    cmd_list = []
+
+    # get folder path
+    project_base = Path(__file__).parent.parent.parent
+    config_tools_path = Path(__file__).parent.parent / 'config_tools'
+
+    add_cmd_list(cmd_list, 'bash -c "find -name "*.log" -delete"', config_tools_path)
+    add_cmd_list(cmd_list, 'bash -c "find -name "*.whl" -delete"', config_tools_path)
+    add_cmd_list(cmd_list, 'bash -c "find -name "*.egg-info" -prune -exec rm -rf {} \;"', config_tools_path)
+    add_cmd_list(cmd_list, 'bash -c "find -name "node_modules" -prune -exec rm -rf {} \;"', config_tools_path)
+    add_cmd_list(cmd_list, 'bash -c "find -name "build" -prune -exec rm -rf {} \;"', config_tools_path)
+    add_cmd_list(cmd_list, 'bash -c "find -name "target" -prune -exec rm -rf {} \;"', config_tools_path)
+    add_cmd_list(cmd_list, 'bash -c "rm -rf dist"', config_tools_path)
+    add_cmd_list(cmd_list, 'bash -c "python ./configurator/packages/configurator/thirdLib/manager.py clean"', config_tools_path)
```

Comment:
Looks like thirdLib/manager.py has a dependency that's not installed in the CI system:
```
Traceback (most recent call last):
  File "./configurator/packages/configurator/thirdLib/manager.py", line 14, in <module>
    from tqdm.auto import tqdm
ModuleNotFoundError: No module named 'tqdm'
cmd(bash -c "python ./configurator/packages/configurator/thirdLib/manager.py clean") run in dir(misc/config_tools) failed and exit
make: *** [Makefile:167: clean] Error 255
```